README.md

```diff
@@ -1,32 +1,45 @@
-# Direct State Transfer (DST) Documentation
+# Perun Documentation
 
-This repository hosts the design documentation for the [dst-go](https://github.com/direct-state-transfer/dst-go) project, which provides a [Go](https://golang.org/) implementation of the [Perun protocol](https://perun.network/).
+This repository hosts the documentation for the Hyperledger Labs project
+[Perun](https://github.com/hyperledger-labs?q=perunl), a
+blockchain-agnostic state channels framework.
 
-## Build
+The documentation is written in
+[reStructuredText](https://www.sphinx-doc.org/en/master/usage/restructuredtext/index.html)
+(or [Markdown](https://www.sphinx-doc.org/en/master/usage/markdown.html)
+of [CommonMark](https://commonmark.org/) flavor) and can be built with
```

Comment:
```suggestion
or [Markdown](https://www.sphinx-doc.org/en/master/usage/markdown.html)
of ([CommonMark](https://commonmark.org/) flavor) and can be built with
```

Reply:
Do you suggest to remove the parentheses or keep them around the markdown flavor? In the latter case it would be:
```suggestion
or [Markdown](https://www.sphinx-doc.org/en/master/usage/markdown.html)
(of [CommonMark](https://commonmark.org/) flavor) and can be built with
```

Reply:
updated PR accordingly.

Reply:
Yes, I think is better.